docs/pages/components/Notice.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Notice
+description: A notification banner - similar to a Toast, but allows for more flexibile positioning and control.
+source: https://github.com/dequelabs/cauldron/blob/develop/packages/react/src/components/Notice/Notice.tsx
+---
+
+import { Notice, Icon } from '@deque/cauldron-react';
+
+```js
+import { Notice } from '@deque/cauldron-react';
+```
+
+## Examples
+
+### Notice Component
+
+The `Notice` component is used to display a notification banner. It is similar to the [Toast](./Toast) component,
+but can be used in more flexible ways.
+
+For example, the `Notice` component can be used to display a notification banner within a [Panel](./Panel) component
+or anywhere else in the DOM without needing to be absolutely/statically positioned.
+
+In addition, the `Notice` component has only limited dynamic functionality built in, allowing developers to
+have full control over the underlying element via its `forwardRef`, while still having useful base styling..
+
+```jsx example
+<Notice type="info" title='Information'>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor sem.
+  Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu condimentum sed.
+</Notice>
+
+<Notice type="caution" title="Caution">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor sem.
+  Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu condimentum sed.
+</Notice>
+```
+
+### Notice With Icon
```

Comment:
Is this the right title for this section? It looks like the heading is what changed here. Additionally, It feels like we should be able to pass in arbitrary headings and have the styles match the expected title style without modifying the heading styles.

Reply:
Hmm, I would argue that they can match the styling by using the same class name. I know we did a few things that were somewhat smart with the Panel, but not sure that makes sense here, especially since my impression of this component was it's intended to allow for a fair amount of flexibility. 

I guess if we were to do that, how do you envision that would work? Cloning the element and adding an arbitrary class during render? Aside from that, what if the title is not intended to be style the way we expect it to? 

IMO, the shared settings beta notification banner is a good example of what I'm referring to.

Reply:
Hmm, not sure I'm fully understanding what you're saying/can't tell if we're saying the same thing or not. I'm going address as much as I can/shouldn't need much discussion and maybe we can talk through a few of the others tomorrow morning?

Reply:
Yeah we can pair on this in the morning and get it knocked out.